src/components/JUPSettingsMenu/JUPSettingsMenu.tsx

```diff
@@ -0,0 +1,105 @@
+import { Settings, Logout, Settings as SettingsIcon, Help as HelpIcon } from "@mui/icons-material";
+import { Tooltip, IconButton, Menu, MenuItem, Avatar, Divider, ListItemIcon, ListItemText, styled } from "@mui/material";
+import SLink from "components/SLink";
+import useAccount from "hooks/useAccount";
+import React, { memo, useCallback, useState } from "react";
+
+const menuProps = {
+  elevation: 0,
+  sx: {
+    overflow: "visible",
+    filter: "drop-shadow(0px 2px 8px rgba(0,0,0,0.32))",
+    mt: 1.5,
+    "& .MuiAvatar-root": {
+      width: 32,
+      height: 32,
+      ml: -0.5,
+      mr: 1,
+    },
+    "&:before": {
+      content: '""',
+      display: "block",
+      position: "absolute",
+      top: 0,
+      right: 14,
+      width: 10,
+      height: 10,
+      bgcolor: "background.paper",
+      transform: "translateY(-50%) rotate(45deg)",
+      zIndex: 0,
+    },
+  },
+};
+
+const JUPSettingsMenu: React.FC = () => {
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const open = Boolean(anchorEl);
+  const { userLogout } = useAccount();
+
+  const handleClick = useCallback((event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+  }, []);
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  return (
+    <>
+      <Tooltip title="Account settings">
+        <IconButton onClick={handleClick} color="primary">
+          <SettingsIcon></SettingsIcon>
+        </IconButton>
+      </Tooltip>
+      <Menu
+        anchorEl={anchorEl}
+        open={open}
+        onClose={handleClose}
+        onClick={handleClose}
+        PaperProps={menuProps}
+        transformOrigin={{
+          horizontal: "right",
+          vertical: "top",
+        }}
+        anchorOrigin={{
+          horizontal: "right",
+          vertical: "bottom",
+        }}
+      >
+        <MenuItem>
```

Comment:
if this menu is going to grow quite large, we can make a task to create a simple array of objects which maps to the proper react jsx to simplify how this code looks. i'm suggesting it as a future task because it seems to me the menu would be a simple pattern repeated, with the odd divider placed between certain categories.

Reply:
I don't expect to have to add many (if any) more items to the menu because I anticipate most additional functionality from the menu to be done when opening a dialog for "user preferences" or similar. So for now I don't think it's an issue, but in time it could be if I decide to tuck more stuff in there.